### PR TITLE
Auto-update main branch and trigger dev release when dependencies are updated

### PR DESCRIPTION
**Pull request by XolphinMartijn**

### CODE REVIEW

File: .github/workflows/dev-build-on-linter-commit.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+name: Scheduled check and build on new zlint commits
```
Comment (robstradling) [inline .github/workflows/dev-build-on-linter-commit.yml:1]: This isn't specific to Zlint.  I suggest:
```suggestion
name: Scheduled check and build on dependency updates
```

File: .github/workflows/dev-build-on-linter-commit.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+on:
+  schedule:
+    # Run Hourly
+    - cron: '0 * * * *'
+  workflow_dispatch:
+  
+jobs:
+  check_release:
+    runs-on: ubuntu-latest
+    name: Daily check for new zlint release
```
Comment (robstradling) [inline .github/workflows/dev-build-on-linter-commit.yml:12]: s/Daily/Hourly/ ?

```suggestion
    name: Hourly check for dependency updates
```

File: .github/workflows/dev-build-on-linter-commit.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    steps:
+    - name: Checkout this repo
+      uses: actions/checkout@v4
+
+    - name: Setup git
+      run: |
+          git config --global user.email "gh@pkimet.al"
+          git config --global user.name "PKIMetal"
```
Comment (robstradling) [inline .github/workflows/dev-build-on-linter-commit.yml:21]:
```suggestion
          git config --global user.name "pkimetal"
```

The project's name is "pkimetal".  No capitals.  No whitespace.  :-)

**Review by robstradling — CHANGES_REQUESTED** [review]
Please also rename "dev-build-on-linter-commit.yml" to "dev-build-on-dependency-update.yml"

**Comment by XolphinMartijn** [comment]
@robstradling  changes addressed

**Review by robstradling — APPROVED** [review]

**Pull request merged**